src/utils/token.ts

```diff
@@ -0,0 +1,42 @@
+import { BigSource } from 'big.js';
```

Comment:
copied from web app, i believe we want to replace all the duplicate code in web app and use sdk there instead, right?

Reply:
eventually yes, rn not so urgent